cli.js

```diff
@@ -56,10 +74,44 @@ function run(file) {
 			return;
 		}
 
-		require(file);
+		var options = {
+			env: assign({}, process.env, {AVA_FORK: 1}),
+			silent: true
+		};
+
+		files++;
+
+		var babel = join(__dirname, 'babel.js');
+
+		var ps = fork(babel, [file], options);
+
+		ps.on('message', test);
+		ps.on('close', exit);
```

Comment:
I think all the fork logic should be outside `cli.js`, in `index.js` or maybe a separate `run.js`/`fork.js` file or something. Separation of concerns and all. This way the CLI could just use a promise interface and get back the result when done without having to know at all that there's forking going on behind the scenes.